main.md

```diff
@@ -737,29 +750,30 @@ The verifier MUST always check the SD-JWT signature to ensure that the SD-JWT
 has not been tampered with since its issuance. If the signature on the SD-JWT
 cannot be verified, the SD-JWT MUST be rejected. 
 
-## Entropy and Uniqueness of the salt
+## Entropy and Uniqueness of the random value
```

Comment:
I wonder if it is clear what we mean with "random value" here. "Salt" was clear because it refers to a specific concept, "random value" seems more arbitrary and leaves room for interpretation. I wonder if we should keep "salt" in the text and just say that in the case of an HMAC, salt becomes the key.

Reply:
reverted to salt. 
need to clarify, it can be key